import java.util.Scanner;

public class OddEvenNumbers {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        // Take input from user
        System.out.print("Enter a natural number: ");
        int number = sc.nextInt();

        // Check if number is natural
        if (number <= 0) {
            System.out.println("Please enter a valid natural number.");
        } 
        else {
            // Loop from 1 to number
            for (int i = 1; i <= number; i++) {

                if (i % 2 == 0) {
                    System.out.println(i + " is Even");
                } 
                else {
                    System.out.println(i + " is Odd");
                }

            }
        }

        sc.close();
    }
}
